### DISCUSSION TITLE
Playing and Pausing

### DISCUSSION THREAD

**Discussion by velmiss** [mkb79/Audible#112] [Q&A]
Hello I've been looking trough the docs but can't find it and want to make sure. Is it possible to play and pause a book playing on a different device than the py script using this?  And if it is possible, how?

**Comment by mkb79** [comment]
Hi. I don't have found an option to pause or resume an audiobook using the API on another device. The only thing you can do is to stream an audiobook using my Audible package and control the media player with Python scripts.

**Reply by velmiss** [reply]
Thank you for the fast answer! I will have to do that it seems
